lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1758,7 +1758,7 @@
           },
           {
             "impact": "critical",
-            "html": "<img src=\"blob:http://localhost:10200/54001a08-60c6-4ea1-aeec-ca1831fd81f1\">",
+            "html": "<img src=\"blob:http://localhost:10200/65381135-47e9-4deb-9f84-bfc18dbf2534\">",
```

Comment:
These urls are random for each run.

Still glad to have done the rebaseline by itself, otherwise it wouldn't be clear that there is no difference in this version from 4.1.1.